ddtrace/debugging/_debugger.py

```diff
@@ -412,13 +426,28 @@ def _dd_debugger_hook(self, probe: Probe) -> None:
                 log.error("Unsupported probe type: %r", type(probe))
                 return
 
-            signal.line()
+            scope = ChainMap(actual_frame.f_locals, actual_frame.f_globals)
```

Comment:
would love to see this also get moved under SignalContext - so there is only one place in our code that deal with those Mixin-s, what do you think?

Reply:
Hmm but we don't need a context for a line probe 🤔